index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- stylesheet -->
+		<link rel="stylesheet" href="style.css" />
+		<!-- fontawesome -->
+			<script src="https://kit.fontawesome.com/d31edfa6bf.js" crossorigin="anonymous"></script>
+		<!-- google font -->
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=M+PLUS+Rounded+1c:wght@100;300;400;500;700&family=Poppins:wght@300;400;500;700&display=swap"
+			rel="stylesheet"
+		/>
+		<title>Portfolio Website</title>
+	</head>
+	<body>
+		<div class="intro-container">
+			<header>
+				<h1>Victor</h1>
+				<div class="hamburger">
+					<i class="fa fa-bars fa-1x"></i>
+				</div>
+			</header>
+			<section class="intro">
+				<h2
+					>I’m Victor <br />
+					Glad to see <br />
+					you!</h2
+				>
+				<p
+					>I’m a software developer! I can help you build a product , feature or website Look
+					through some of my work and experience! If you like what you see and have a project you
+					need coded, don’t hestiate to contact me.</p
+				>
+        <p id="connect">LETS CONNECT</p>
+				<div class="socials">
+					<a href="https://twitter.com/Vicky_marz" class="icon"
+						><i class="fab fa-twitter fa-x"></i
+					></a>
+					<a href="https://www.linkedin.com/in/okoroji-victor-ebube-8791741a0" class="icon"
+						><i class="fab fa-linkedin fa-fw"></i
+					></a>
+					<a href="https://medium.com/@okorojiebube2" class="icon"
+						><i class="fab fa-medium-m fa-fw"></i
+					></a>
+					<a href="https://github.com/vickymarz" class="icon"
+						><i class="fab fa-github fa-fw"></i
+					></a>
+					<a href="#" class="icon"><i class="fa fa-angellist fa-fw"></i></a>
+				</div>
```

Comment:
- [ ] Good job adding the social icons in an `a` tag 🔥. Please consider using a `ul` tag to wrap all the social icons and use `li` tag to wrap each of the social icons instead of a `div` tag. This is to improve the semantic structure of your page and improve the code structure too. See my suggestion below:

```suggestion
				<ul class="socials">
				     <li>
					<a href="https://twitter.com/Vicky_marz" class="icon"
						><i class="fab fa-twitter fa-x"></i
					></a>
				    </li>
				     <li>
					<a href="https://www.linkedin.com/in/okoroji-victor-ebube-8791741a0" class="icon"
						><i class="fab fa-linkedin fa-fw"></i
					></a>
				    </li>
				    <li>
					<a href="https://medium.com/@okorojiebube2" class="icon"
						><i class="fab fa-medium-m fa-fw"></i
					></a>
				    </li>
				    <li>
					<a href="https://github.com/vickymarz" class="icon"
						><i class="fab fa-github fa-fw"></i
					></a>
				    </li>
				    <li>
					<a href="#" class="icon"><i class="fa fa-angellist fa-fw"></i></a>
				    </li>
				</ul>
```